gatsby_fp/gatsby-config.js

```diff
@@ -7,11 +7,88 @@ module.exports = {
   },
   plugins: [
     {
-      resolve: "gatsby-source-multi-api-FORKED",
+      resolve: "gatsby-source-custom-api",
       options: {
-        apis: [
-          `https://${process.env.PEOPLEAPI_USER}:${process.env.PEOPLEAPI_PASS}@${process.env.PEOPLEAPI_HOST}`,
-        ],
+        url: `https://${process.env.PEOPLEAPI_USER}:${process.env.PEOPLEAPI_PASS}@${process.env.PEOPLEAPI_HOST}`,
+        rootKey: 'profiles',
+        schemas: {
+          profiles: `
+          pf_username: String
+          appointment: String
+          bio: String
+          building: String
+          consult_service: String
+          experience: String
+          grant_contract: String
+          honor_award: String
+          innovate_enterpreneur: String
+          patent_invention: String
+          pf_email: String
+          pf_work_fax: String
+          pf_first_name: String
+          pf_last_name: String
+          pf_work_phone: String
+          pf_title: String
+          pf_username: String
+          website: String
+          research: String
+          room: String
+          photo_base64: String
+          notable_courses: String
+          school: String
+          ses_department: String
+          service_university: [service_university]
+          service_professional: [service_professional]
+          education: [education]
+          member: [member]
+          intellcont: [intellcont]
+        `,
```

Comment:
Inconsistent indentation (this backtick should have the same amount of indentation as line 15)

Reply:
I fixed it, it's not the problem